app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [marks, my_favourite_films, collection_of_coins],
+    "immutable": [name, profile_info, one_is_a_prime_number, lucky_number, pi]
+}
```

Comment:
Please fix the problem with flake8, don't forget the empty line at the end of the file.